packages/web-react/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,36 @@
+import classNames from 'classnames';
+import React from 'react';
+import { useStyleProps } from '../../hooks/styleProps';
+import { useClassNamePrefix, useIcon } from '../../hooks';
+import { IconProps } from '../../types';
+
+const defaultProps = {
+  size: 24,
+};
+
+export const Icon = ({ name, title, size, ...restProps }: IconProps): JSX.Element => {
+  const iconClass = useClassNamePrefix('Icon');
+  const icon = useIcon(name);
+  const { styleProps, props: otherProps } = useStyleProps(restProps);
+
+  return (
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore: Incompatible HTMLElement and SVGSVGElement
+    <svg
+      viewBox={`0 0 ${size} ${size}`}
+      fill="currentColor"
+      width={size}
+      height={size}
+      title={title}
```

Comment:
Is it possible to add the title as `<title>` children into `<svg>`? It would be consistent with Twig implementation and it would work properly then. If it would be the first child (before the dangerous HTML) it would even override any title in the HTML and it would work.

Reply:
Ok, sure, added.